### DISCUSSION TITLE
How to change the parent theme?

### DISCUSSION THREAD

**Discussion by desquivel-vgw** [keycloakify/keycloakify#622] [Q&A]
Hi, I was wondering how could I change the parent theme used in my Theme generated by Keycloakify?

If I create a theme in Keycloack (without keycloakify) , I can do this by just changing the theme.properties and writing something like:

`"parent=YOUR_PARENT_THEME"`

Oficial doc: https://www.keycloak.org/docs/latest/server_development/#creating-a-theme

Is there any work around? Maybe a postbuild action? Any suggestion is more than welcome.

**Comment by garronej** [comment]
Hello @desquivel-vgw,

Could you clarify what you're aiming to achieve? While it is possible to set a parent theme, doing so won't have any impact. Keycloakify themes are fully self-contained, ensuring a consistent look and feel across all Keycloak versions.

If your goal is to set the `parent` to `base` to avoid the default styles, you can accomplish this by setting `doUseDefaultCss` to `false`. You can find more details in the [documentation](https://docs.keycloakify.dev/customization-strategies/css-level-customization/removing-the-default-styles#removing-all-the-default-styles-at-once).

If you're looking to extend a theme because it includes additional pages not found in the default Keycloak set, you might want to check out [this documentation page](https://docs.keycloakify.dev/styling-custom-extension-page).

Let me know if this answers your question.
